deploy.go

```diff
@@ -95,7 +95,7 @@ func (p *deploycmd) scan(c *cli.Context) error {
 
 	err = filepath.Walk(wd, func(path string, info os.FileInfo, err error) error {
 		if path != wd && info.IsDir() {
-			return filepath.SkipDir
+			return nil
```

Comment:
this is the only thing i'm not sure about. this means where previously we would skip over directories that were not the directory we were looking for (if in fact, the path we were looking for ended up being a directory), now we will walk the contents of that directory. maybe this is what was broken (makes sense), fwiw it would still make sense to skip entire directory contents (save a lot of i/o) only in the case that the path we're seeking for and the current path do not override. e.g.

```
a/b
a/c

path=a/c/foo
```

now we will rummage through the contents of `a/b` even though we are certain that the file/dir we are looking for is not there. if somebody is doing this in their `Downloads` folder or something then it could be expensive.